DesktopUI2/DesktopUI2/ViewModels/HomeViewModel.cs

```diff
@@ -290,6 +290,7 @@ internal void UpdateSelectedStream()
       }
       catch (Exception ex)
       {
+        //FIXME: This branch can't ever get hit right?
         Log.Error(ex, ex.Message);
```

Comment:
Add a message template to this pls, otherwise its very hard to trace back from the error message...